### ISSUE TITLE
model

### ISSUE THREAD

**Issue by ez0012** [ez0012/Kaggle#1]
import torch
import torch.nn as nn
from transformers import BertModel

class CustomNLPModel(nn.Module):
    def __init__(self, hidden_dim):
        super(CustomNLPModel, self).__init__()
        self.bert = BertModel.from_pretrained('bert-base-uncased')
        self.highway = nn.Highway()

        self.mlp = nn.Sequential(
            nn.Linear(768, hidden_dim),
            nn.ReLU(),
            nn.Linear(hidden_dim, 2)
        )

    def forward(self, input_ids, attention_mask):
        with torch.no_grad():
            bert_output = self.bert(input_ids=input_ids, attention_mask=attention_mask)

        mean_pooled = bert_output.last_hidden_state.mean(dim=1)
        return self.mlp(mean_pooled)
